packages/core/src/App/Containers/RealAccountSignup/personal-details.jsx

```diff
@@ -199,33 +206,85 @@ class PersonalDetails extends React.Component {
-                                                                            true
-                                                                        )
-                                                                    }
-                                                                />
+                                                                <div className='details-form__tax'>
+                                                                    <Autocomplete
+                                                                        {...field}
+                                                                        data-lpignore='true'
+                                                                        autoComplete='off' // prevent chrome autocomplete
+                                                                        type='text'
+                                                                        label={localize('Tax residence')}
+                                                                        error={
+                                                                            touched.tax_residence &&
+                                                                            errors.tax_residence
+                                                                        }
+                                                                        list_items={this.props.residence_list}
+                                                                        onItemSelection={({ value, text }) =>
+                                                                            setFieldValue(
+                                                                                'tax_residence',
+                                                                                value ? text : '',
+                                                                                true
+                                                                            )
+                                                                        }
+                                                                    />
+                                                                    <Popover
+                                                                        alignment='right'
+                                                                        icon='info'
+                                                                        message={localize(
+                                                                            'Tax residence, also known as fiscal residency or redisence for tax purposes, is an important concept for all taxpayers living and working abroad. It determines the tax liabilities that the individual has to beer within a particular country (jurisdiction).'
+                                                                        )}
+                                                                        zIndex={9999999}
```

Comment:
(throughout) What is the minimum amount we can use for `zIndex` to work?

Reply:
Values `>=`  9999 
_9999_ is the value of modal zIndex.